File: repos_estudiantes_ALVAREZ_primera_entrega/CALLE_CASTILLO_CRISTHIAN_JAVIER/PRACTICA1-PROGRAMACION-II/PRACTICA INF-121/PROGRAMACION ORIENTADA A OBJETOS/EJERCICIO 1/Coche.py
class Coche:
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad

    def acelerar(self):
        self.velocidad += 10

    def frenar(self):
        self.velocidad -= 5

    def mostrar_velocidad(self):
        print(f"{self.marca} {self.modelo} va a {self.velocidad} km/h")

coche1 = Coche("Toyota", "Corolla", 0)
coche2 = Coche("Ford", "Bronco", 20)

coche1.acelerar()
coche1.acelerar()
coche1.frenar()
coche1.mostrar_velocidad()

coche2.frenar()
coche2.acelerar()
coche2.mostrar_velocidad()

File: repos_estudiantes_ALVAREZ_primera_entrega/CALLE_CASTILLO_CRISTHIAN_JAVIER/PRACTICA1-PROGRAMACION-II/PRACTICA INF-121/PROGRAMACION ORIENTADA A OBJETOS/EJERCICIO 3/Coche.java
public class Coche {
    String marca;
    String modelo;
    int velocidad;

    public Coche(String marca, String modelo, int velocidad) {
        this.marca = marca;
        this.modelo = modelo;
        this.velocidad = velocidad;
    }

    public void acelerar() {
        velocidad += 10;
    }

    public void frenar() {
        velocidad -= 5;
    }

    public void mostrarVelocidad() {
        System.out.println(marca + " " + modelo + " va a " + velocidad + " km/h");
    }

    public static void main(String[] args) {
        Coche coche1 = new Coche("Toyota", "Corolla", 0);
        Coche coche2 = new Coche("Ford", "Fiesta", 20);

        coche1.acelerar();
        coche1.acelerar();
        coche1.frenar();
        coche1.mostrarVelocidad();

        coche2.frenar();
        coche2.acelerar();
        coche2.mostrarVelocidad();
    }
}

